lib/worker.js

```diff
@@ -144,14 +141,13 @@ class Worker extends BaseService {
             }, 5000);
         });
 
-        // Metrics reporter
-        this._metrics = makeStatsD(this.config.metrics, this._logger);
-
+        // Metrics reporting
+        this.metrics = makeMetrics(this.config.metrics, this._logger);
```

Comment:
Why would we remove the `_` prefix from the internal property name?

Reply:
I suppose there is no good reason to rename it.  I'll fix that up.